packages/tsx-editor/src/components/Editor.tsx

```diff
@@ -23,10 +17,22 @@ export const Editor: React.FunctionComponent<IEditorProps> = (props: IEditorProp
       experimentalDecorators: true,
       preserveConstEnums: true,
       outDir: 'lib',
-      module: monaco.languages.typescript.ModuleKind.CommonJS,
+      module: monaco.languages.typescript.ModuleKind.ESNext,
       lib: ['es5', 'dom']
     });
 
+    monaco.languages.typescript.typescriptDefaults.setDiagnosticsOptions({ noSemanticValidation: true });
```

Comment:
What does this do?

Reply:
This disables the red lines. I disabled them while nishi works on it.